sdk/python/foundation-models/system/inference/text-generation/llama-files/score/score.py

```diff
@@ -460,7 +460,9 @@ def analyze_text_async(text):
 
 def analyze_text(text):
     # Chunk text
-    print(f"Analyzing ...")
+    print("Analyzing ...")
+    if (not text) or (not text.strip()):
+        return 0
```

Comment:
Would this make return same empty response as final output?

Reply:
This will leave string field in request/response untouched.
So if model return empty, then endpoint return empty as well.